config/Config-build.in

```diff
@@ -114,6 +114,14 @@ menu "Global build settings"
 		  This makes file checksums part of package metadata. It increases size
 		  but provides you with pkg_check command to check for flash corruptions.
 
+	config IPK_REMOVE_MAINTAINER
+		bool
+		prompt "Remove packages maitainer information on the target"
+		help
+		  Attackers know, that developers have access to source code of the device
```

Comment:
Don't you think attackers also know, that the source code contain maintainer information? If you try to malicoiously get commit access to source code aka a repository, you likely look at it before (e.g. to understand its impact). In that case one easily obtain that information anyway.

Reply:
That's not quite right. This is not about the creator of the source code but about the maintainer of the package at OpenWrt.
I know that you can get the information in other ways, but you don't have to make it too easy?
The creator of the OpenWrt package usually does not know the internals of the upstream package he maintains, but he can add patches, which is an attack vector if you put the OpenWrt package maintainer under pressure.

Reply:
Sounds a bit like "security by obscurity" to me. Distributions make these information very easy accessible, like [Debian](https://packages.debian.org/sid/devel/tmate), [Alpine](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/tmate) or even [OpenWrt](https://openwrt.org/packages/pkgdata/tmate).

Reply:
Thanks for your feedback. Absolutely right, I agree with you that is "security by obscurity" if that's what you want to call it. The original question from my side was actually do we need the information on the rootfs target?
Isn't it enough that we have it in the repo? Why not always remove it on the rootfs target?